src/Integration/LocalServices/IBindingRequiredIndicator.cs

```diff
@@ -0,0 +1,55 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Linq;
+using SonarLint.VisualStudio.Integration.Resources;
+
+namespace SonarLint.VisualStudio.Integration
+{
+    interface IBindingRequiredIndicator
+    {
+        bool IsBindingRequired();
+    }
+
+    internal class BindingRequiredIndicator : IBindingRequiredIndicator
+    {
+        private readonly IUnboundProjectFinder unboundProjectFinder;
+        private readonly ILogger logger;
+
+        public BindingRequiredIndicator(IUnboundProjectFinder unboundProjectFinder, ILogger logger)
```

Comment:
UnboundProjectFinder (and the interface) are also currently in this project instead of Core, so I placed them here side by side.